redfish-core/lib/log_services.hpp

```diff
@@ -4071,32 +4071,44 @@ inline void handleLogServicesDumpEntryComputerSystemDelete(
         return;
     }
 
-    if (boost::starts_with(dumpId, "System"))
+    std::size_t pos = dumpEntryId.find_first_of('_');
+    if (pos == std::string::npos || (pos + 1) >= dumpEntryId.length())
+    {
+        // Requested ID is invalid
+        messages::invalidObject(
+            asyncResp->res,
+            crow::utility::urlFromPieces("redfish", "v1", "Systems", chassisId,
+                                         "EthernetInterfaces", dumpEntryId));
+        return;
+    }
+
+    const std::string& dumpId = dumpEntryId.substr(pos + 1);
+    if (boost::starts_with(dumpEntryId, "System"))
     {
         deleteDumpEntry(asyncResp, dumpId, "System");
     }
-    else if (boost::starts_with(dumpId, "Resource"))
+    else if (boost::starts_with(dumpEntryId, "Resource"))
     {
         deleteDumpEntry(asyncResp, dumpId, "Resource");
     }
-    else if (boost::starts_with(dumpId, "Hostboot"))
+    else if (boost::starts_with(dumpEntryId, "Hostboot"))
     {
         deleteDumpEntry(asyncResp, dumpId, "Hostboot");
     }
-    else if (boost::starts_with(dumpId, "Hardware"))
+    else if (boost::starts_with(dumpEntryId, "Hardware"))
     {
         deleteDumpEntry(asyncResp, dumpId, "Hardware");
     }
-    else if (boost::starts_with(dumpId, "SBE"))
+    else if (boost::starts_with(dumpEntryId, "SBE"))
     {
         deleteDumpEntry(asyncResp, dumpId, "SBE");
     }
     else
     {
-        messages::invalidObject(asyncResp->res,
-                                crow::utility::urlFromPieces(
-                                    "redfish", "v1", "Systems", "system",
-                                    "LogServices", "Dump", "Entries", dumpId));
+        messages::invalidObject(
+            asyncResp->res, crow::utility::urlFromPieces(
+                                "redfish", "v1", "Systems", chassisId,
```

Comment:
chassisId here doesn't look correct

Reply:
The chassisId will anyways be "system" right?
Because we are checking if chassisId is system and if not, we are throwing an error.

Reply:
Can be fix this var name to be correct so systemId ?